### PR TITLE
HiPAC2023 competition

### PR DESCRIPTION
**Pull request by shirou1217**

### CODE REVIEW

File: src/content/competitions/HiPAC2023.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+We are deeply grateful for the support provided by the National Center for High-performance Computing. Access to their Taiwania2 and Taiwania3 clusters was crucial for testing and refining our solutions. 
+
+Participating in the HPC competition was a challenging yet rewarding experience that enriched our technical skills, fostered teamwork, and deepened our appreciation for the power of high-performance computing. We are proud of our achievements and look forward to leveraging this experience in our future endeavors. This competition was not just about winning but about learning, growing, and pushing the limits of what we can achieve with technology.
+
+Competition Winners Announcement: [link](https://event.nchc.org.tw/2023/hipac/awards.html)
+
+- Team-2 Members: SHIH-JOU CHENG, WEI-CHIH HUANG, SSU-CHENG Lai, WEI-PO LIN, TUNG-YU HSIEH, CHIH-YU HSIEH
```
Comment (William-Mou) [inline src/content/competitions/HiPAC2023.md]: maybe
"SSU-CHENG Lai", should be replaced with "SSU-CHENG LAI"

File: src/content/competitions/HiPAC2023.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Championship and second place at HiPAC 2023 competition"
+description: "NTHU Team-2 and Team-ZY secure the champion and second place respectfully at 2nd HIGH PERFORMANCE APPLICATION COMPETITION 2023"
+pubDate: "Aug 10 2023"
+heroImage: "/HiPAC-2023-banner.jpg"
+---
+Our student team from National TsingHua University's Computer Science department have secured the champion of 2nd HIGH PERFORMANCE APPLICATION COMPETITION again, with a second place also taken done by us!
```
Comment (William-Mou) [inline src/content/competitions/HiPAC2023.md]: "National TsingHua University" should be replaced with "National Tsing Hua University"
The name of Tsinghua University in Beijing should be written together as one word.

**Review by William-Mou — CHANGES_REQUESTED** [review]
some minor typo, please fix it. tks~

File: src/content/competitions/HiPAC2023.md (hunk 2 above)
Comment (shirou1217) [reply src/content/competitions/HiPAC2023.md]: fixed it!

File: src/content/competitions/HiPAC2023.md (hunk 1 above)
Comment (shirou1217) [reply src/content/competitions/HiPAC2023.md]: fixed it!

**Review by William-Mou — APPROVED** [review]
LGTM!

**Pull request merged**